MPC/config/opendds_util.mpb

```diff
@@ -3,4 +3,6 @@ project: dds_taolib {
   libs += OpenDDS_Util
   includes += $(DDS_ROOT)
   libpaths += $(DDS_ROOT)/lib
+  macros   += NOTAO
```

Comment:
`OPENDDS_NOTAO`?

Reply:
Nevermind, the library is just for `opendds_idl` right?

Reply:
Right.  Eventually, I think this will get renamed but I think we'll have a better idea of what that is in the future.